packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -318,7 +316,11 @@ export class PeerManager {
     // if the sequence number is unknown update the peer's metadata
     const metadata = this.connectedPeers.get(peer.toString())?.metadata;
     if (!metadata || metadata.seqNumber < seqNumber) {
+      // re-status happens at end of requestMetadata to keep peer info up to date
       void this.requestMetadata(peer);
+    } else if (this.clock.currentEpoch > this.config.FULU_FORK_EPOCH) {
+      // if metadata didn't change just re-status when post-fulu to check for earliestAvailableSlot updates
+      void this.requestStatus(peer, this.statusCache.get());
```

Comment:
I feel it's too much to request status per ping. The idea of ping message is the most lightweight p2p message so that we can do that with the most frequency. With this change we also make status message the most frequency postfulu

the reason we do that is to get updated `earliestAvailableSlot` but that does not change frequently for a lot of peers. We already do status update for every 5 minutes, that should be enough frequency

the most beneficial consumer of `earliestAvailableSlot` is range sync but that only happens when we start the node for a short period of time